lib/chart/animated_line_chart.dart

```diff
@@ -270,8 +383,12 @@ class ChartPainter extends CustomPainter {
     });
 
     highlights.forEach((highlight) {
-      canvas.drawCircle(Offset(highlight.chartPoint.x, highlight.chartPoint.y),
-          5, _linePainter);
+      if (_chart.lines[index].isMarkerLine != true) {
```

Comment:
isMarkerLine is a bool. Something like  `if(!_chart.lines[index].isMarkerLine)` would do. Do you expect that isMarkerLine could be null ? In case it can be then may be have some null check as well or start with a default value for the field....